### PR TITLE
skeptipay landing structure

### PR DESCRIPTION
**Pull request by KwaneleNdenze**

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  end
+
+  def about
+  end
+
+  def blog
+  end
+
+  def contact
+  end
```
Comment (thatosmk) [inline app/controllers/pages_controller.rb:12]: Keep spaces between your class start and class end
```suggestion

  def index
  end

  def about
  end

  def blog
  end

  def contact
  end
  
```

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -4,12 +4,167 @@
...
 
   <body>
+  <header id="header" class="fixed-top d-flex align-items-center">
+    <div class="container d-flex align-items-center">
+
+      <img src="Sailor/assets/img/SkeptiPayLogo2.png" class="img-fluid" width="50px" height="50px" alt=""><div style="padding-left:10px;"><h2 class="logo me-auto"><%= link_to 'SkeptiPay', root_path %></h2></div>
+      <!-- Uncomment below if you prefer to use an image logo -->
+      <a href="index.html" class="logo me-auto"><img src="Sailor/assets/img/logo.png" alt="" class="img-fluid"></a>
+
+      <nav id="navbar" class="navbar">
```
Comment (thatosmk) [inline app/views/layouts/application.html.erb:46]: you can create partials;
```ruby
<%= render 'layouts/navbar' %> # this is a rails partial

# then create the file layouts/_navbar.html.erb and place the nav in there
```

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -4,12 +4,167 @@
...
+          
+        </ul>
+        <i class="bi bi-list mobile-nav-toggle"></i>
+      </nav><!-- .navbar -->
+
+    </div>
+  </header><!-- End Header -->
     <%= yield %>
+    <!-- ======= Footer ======= -->
+  <footer id="footer">
```
Comment (thatosmk) [inline app/views/layouts/application.html.erb:82]: same thing with the footer, make it a partial

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -4,12 +4,167 @@
...
+
+  <a href="#" class="back-to-top d-flex align-items-center justify-content-center"><i class="bi bi-arrow-up-short"></i></a>
+
+  <!-- Sailor/Vendor JS Files -->
+  <script src="Sailor/assets/vendor/bootstrap/js/bootstrap.bundle.min.js"></script>
+  <script src="Sailor/assets/vendor/glightbox/js/glightbox.min.js"></script>
+  <script src="Sailor/assets/vendor/isotope-layout/isotope.pkgd.min.js"></script>
+  <script src="Sailor/assets/vendor/swiper/swiper-bundle.min.js"></script>
+  <script src="Sailor/assets/vendor/waypoints/noframework.waypoints.js"></script>
+  <script src="Sailor/assets/vendor/php-email-form/validate.js"></script>
```
Comment (thatosmk) [inline app/views/layouts/application.html.erb:164]: which one of these will you use?

File: app/views/pages/blog.html.erb [hunk 5]
```diff
@@ -0,0 +1,241 @@
+<main id="main">
+
```
Comment (thatosmk) [inline app/views/pages/blog.html.erb:2]: Maybe this should just have a header for now

File: app/views/pages/about.html.erb [hunk 6]
```diff
@@ -0,0 +1,425 @@
+<main id="main">
+
+    <!-- ======= Breadcrumbs ======= -->
+    <section id="breadcrumbs" class="breadcrumbs">
+      <div class="container">
+
+        <div class="d-flex justify-content-between align-items-center">
+          <h2>About</h2>
+          <ol>
+            <li><a href="index.html">Home</a></li>
```
Comment (thatosmk) [inline app/views/pages/about.html.erb:10]: things like this you won't need...

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (KwaneleNdenze) [reply app/controllers/pages_controller.rb:12]: I've made this change

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (KwaneleNdenze) [reply app/views/layouts/application.html.erb:46]: I've created the partial.

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (KwaneleNdenze) [reply app/views/layouts/application.html.erb:82]: Made the footer partial.

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (KwaneleNdenze) [reply app/views/layouts/application.html.erb:164]: I've removed the last script (php-email-form/validate.js). The rest I think I'll need to use, removing the bootstrap scripts distorts the views.

**Comment by thatosmk** [comment]
love the new changes you made, let's chat through them during standup tomorrow. Great work dude.

File: .gitignore [hunk 7]
```diff
@@ -29,3 +29,8 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+/public/packs
+/public/packs-test
+/node_modules
+yarn-debug.log*
+.yarn-integrity
```
Comment (thatosmk) [inline .gitignore:36]: Can you try this as well?
```suggestion
.yarn-integrity
**/.DS_Store
```

Comment (KwaneleNdenze) [reply .gitignore:36]: Added the **/.DS_Store line to the .gitignore file.

**Review by thatosmk — APPROVED** [review]

**Pull request merged**